Add private sourcemaps doc

mmcdaris opened the pull request:
Documents our private sourcemaps feature.

Provides instructions on how to generate and upload sourcemaps to Airbrake.

vmihailenco commented on jekyll/_docs/installing-airbrake/private-sourcemaps.md:
````diff
@@ -0,0 +1,77 @@
+of the sourcemap.
+
+```
+js //# sourceMappingURL=app.min.js.map
````
`js` is on wrong line

vmihailenco commented on jekyll/_docs/installing-airbrake/private-sourcemaps.md:
````diff
@@ -0,0 +1,77 @@
+[webpack](https://webpack.js.org/configuration/devtool/#production).  Here is an
+example to minify and create a sourcemap for an `app.js` file:
+
+```js
````
this is shell I guess

vmihailenco commented:
Overall looks good :+1:

mmcdaris commented:
Thanks @vmihailenco, PTAL. I've pushed changes addressing the points you brought up, put a note at the top about this being a premium feature, and moved this doc to the features section.

thompiler commented on jekyll/_docs/features/private-sourcemaps.md:
```diff
@@ -0,0 +1,78 @@
+**Private sourcemaps are a premium feature available on the Business Plus and
+Enterprise plans.**
+
+This guide assumes you have already [installed Airbrake in you JavaScript
```
Typo: `you` -> `your`
>installed Airbrake in **your** JavaScript

thompiler commented on jekyll/_docs/features/private-sourcemaps.md:
```diff
@@ -0,0 +1,78 @@
+
+## Uploading sourcemaps
+
+Contining on with our above example, the next steps assume the minified version
```
Typo: `Contining` -> `Continuing`

mmcdaris replied:
Thanks @thompiler, I've pushed a typo fix.

mmcdaris commented:
Thanks for the reviews!
